core/src/main/java/org/elasticsearch/action/delete/DeleteRequest.java

```diff
@@ -178,6 +181,9 @@ public long version() {
     }
 
     public DeleteRequest versionType(VersionType versionType) {
+        if (versionType == VersionType.FORCE) {
```

Comment:
I wondering if we should do it on validate? feels more natural there, as we won't need to wonder if there if we got all the paths to set version types